### ISSUE TITLE
gcc, gmp: segmentation fault with handling float number

### ISSUE THREAD

**Issue by mattn** [Alexpux/MSYS2-packages#452]
Sorry, I'm not well known about msys2 bug-reporting. Currently, the version of gmp is 6.1.0 but it seems having a bug. Try below.

```
$ LANG=C gcc -c float.c
float.c: In function 'foo':
float.c:2:3: internal compiler error: Segmentation fault
   100.0;
   ^
Please submit a full bug report,
with preprocessed source if appropriate.
See <http://gcc.gnu.org/bugs.html> for instructions.
```
I did try to downgrade gmp into 6.0.0. And I didn't see the segfault. So, I'm thinking package of gmp should be downgraded on msys2 package system.Or this repository have better to provide PKGBUILD of 6.0.0. How do you think?

**Comment by mabreedlove** [comment]
This will probably get more attention if you provided the test code you are trying to compile so others have something they can use to reproduce it.

**Comment by mattn** [comment]
This is solved in 6.1.0-1
